### PR TITLE
BUG: Fix wrapping.

### PR DESCRIPTION
**Pull request by jhlegarreta**
Fix wrapping: wrap correct class `itk::StripTsImageFilter` (was
`itk::SkullStrip`).

### CODE REVIEW

**Comment by jhlegarreta** [comment]
Well, not sure how to fix the wrapping error then... @fbudin69500 @thewtex any hint?

**Comment by jhlegarreta** [comment]
@phcerdan ?

File: wrapping/itkStripTsImageFilter.wrap [hunk 1]
```diff
@@ -1,3 +1,8 @@
-itk_wrap_class("itk::SkullStrip" POINTER)
-  itk_wrap_image_filter("${WRAP_ITK_REAL}" 2 2+)
+itk_wrap_class("itk::StripTsImageFilter" POINTER)
+  foreach(d ${ITK_WRAP_IMAGE_DIMS})
+    foreach(t ${WRAP_ITK_REAL})
+      itk_wrap_template("${ITKM_${t}}${d}${ITKM_${t}}${d}${ITKM_${t}}${d}"
+                        "itk::Image< ${ITKT_${t}},${d} >, itk::Image< ${ITKT_${t}}, ${d} >, "itk::Image< ${ITKT_${t}}, ${d} >")
```
Comment (dzenanz) [inline wrapping/itkStripTsImageFilter.wrap]: You have 3 `"` symbols on this line, meaning they are not matched. You probably need to remove the middle one.

Comment (jhlegarreta) [reply wrapping/itkStripTsImageFilter.wrap]: Good catch :eagle: @dzenanz ! Thanks. Let's see if e8d3485 makes it !

**Comment by jhlegarreta** [comment]
Although this solves the previous error, it looks like not all image/filter combinations are available. The first commit in the PR was meant to avoid that, but looks like there's something missing.

**Pull request merged**